lib/DXIL/DxilOperations.cpp

```diff
@@ -5489,7 +5492,8 @@ llvm::Type *OP::GetOverloadType(OpCode opCode, llvm::Function *F) {
   case OpCode::CallShader:
   case OpCode::Pack4x8:
   case OpCode::WaveMatrix_Fill:
-    DXASSERT_NOMSG(FT->getNumParams() > 2);
+    if (FT->getNumParams() <= 2)
+      return Ty;
```

Comment:
Should these return a value, or should they return an error of some sort (nullptr)?

Reply:
Changed to return nullptr.